pkg/protobuf/v2/bin/generate-include-paths

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+go_modules=($(go list -f '{{.Path}}={{.Dir}} ' -m all))
+
+for i in "${go_modules[@]}"
+do
+    path="${i%=*}"
+    dir="${i#*=}"
+    proto_file_count=$(find $dir -type f -iname '*.proto' | wc -l)
+
+    if [[ $proto_file_count -gt 0 ]]; then
```

Comment:
```suggestion
    if find "$dir" -type f -iname '*.proto' > /dev/null; then
```

I'm not really sure this suggestion is any better, but the exit code of `find` can be used in an `if` test directly, eliminating the need to count lines of output.